examples/pnnl-example1.md

```diff
@@ -0,0 +1,13 @@
+# PNNL Example Building 1
+
+This is an example building model provided by Pacific Northwest National Laboratory.
+The building is simplistic, consisting of a single story with 5 rooms (open office, private office, bathroom, kitchenette, corridor).
+The model was created using an alpha version of the PNNL Revit2RDF tool that creates a semantic model from a Revit building information model (BIM) exported via a Speckle (https://speckle.systems/) interface.
+A Beta version of the Revit2RDF tool will tentatively be released via GitHub sometime in 2024.
```

Comment:
```suggestion
```
Is this info important and relevant to users?

Reply:
I think so, in particular for the Pritoni model, as we will have at least 2 models for the same building, created via different workflows, modeling different levels of detail, and using different complimentary ontologies: 1) created by Steve, Parastoo, and myself using TopBraid, modeling Mechanical and Electrical/Lighting - including FunctionBlocks, not modeling space types 2) created by PNNL using the Revit2RDF tool, modeling only Electrical/Lighting - but not (yet) including function blocks, and modeling space types using the REC ontology.

In general, I think its helpful for users to have some understanding of how the model was created and what is in the model prior to reviewing the model. Also, what's the harm in additional detail?

Reply:
I like knowing that this was created as an export from a BIM model, but I'm reluctant to mention a tool until it's actually available somewhere and we can link to it

Reply:
Ok, my main concern was what Gabe mentioned, i.e. linking a tool that doesn't exist in the public domain yet. I like the idea of some sort of lineage description of the models but I think it can just be as simple as what the `Source` was, e.g. BIM, BEM, manual, etc. So maybe a `Source` heading in the MD file would be a nice addition.

Reply:
That makes sense. I'll make some edits, and you can let me know what you think!

Reply:
@MatthewSteen @gtfierro just committed edits to the markdown file. Thoughts?

Reply:
Looks good to me! Thanks for making the change

Reply:
@gtfierro thanks! Will one of you make the merge, or can/should I go ahead and do it?